opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipOverview.js

```diff
@@ -420,6 +431,27 @@ class StixCoreRelationshipContainer extends Component {
                           stixCoreRelationship.description
                         )}
                       </Markdown>
+                      <Typography variant="h3" gutterBottom={true} style={{ marginTop: 20 }}>
+                        {t('Kill chain phases')}
+                      </Typography>
+                      <List>
```

Comment:
Do you think we can have a generic component like StixCoreObjectLabelsView ?
To manage in the same way the kill chain in Tool, Malware and now StixCoreRelationship.

Reply:
right ! I'll do this !

Reply:
right ! I'll do this !